fintrack/choices.py

```diff
@@ -0,0 +1,11 @@
+from django.db import models
+
+
+class TransactionCategoriesChoices(models.TextChoices):
+    Withdraw = "Withdraw", ("Withdraw")
```

Comment:
even after putting the comment with the exact changes, still changes aren't made